tests/e2e/vmgroupmisc_test.go

```diff
@@ -265,3 +265,83 @@ func (s *vgBasicSuite) TestDeleteVMFromVmgroup(c *C) {
 	// Please add step to delete vmgroup after issue 1484 is fixed.
 	misc.LogTestEnd(c.TestName())
 }
+
+// Test steps:
```

Comment:
following would be an issue with the proposed test method design ..

If any of the step fails resulting stop the test execution right away instead it should be broken down into smaller test methods.